schema.graphql

```diff
@@ -485,11 +544,78 @@ type MatchQuestionBuzz {
 """
 Docs group: Match
 """
-type MatchQuestionBonus {
+type BuzzPosition {
+  """
+  The time of the buzz. An ISO 8601-formatted string, with time zone, in the tournament's local time zone. May not be before the start time of reading the relevant question.
+  """
+  time: Date
+
+  """
+  The (`0`-based) index of the word on which the player buzzed.
   """
-  Which bonus number this was. May or may not match the `question_number` of the `MatchQuestion` parent, depending on whether bonuses are paired with tossups or simply read in order (and if the latter, on whether any bonus-earning questions failed to lead to a bonus).
+  word_index: Int
+
+  """
+  The (`0`-based) index of the clue on which the player buzzed. Clues may be defined in the `Question` object.
+  """
+  clue_index: Int
+}
+
+"""
+This object can be reused for every time the same question is read, i.e., it represents a question in general rather than a question being read in a specific match.
+
+Docs group: Match
+"""
+type Question {
+  """
+  The number of the question in the packet.
   """
   question_number: Int!
+  
+  """
+  The type of the question. We generally expect `tossup` or `bonus`, but are not restricting the values since some formats may use other types of questions.
```

Comment:
Let's restrict the values and make this an enum.

Reply:
I was pretty deliberate with this (and explained my decision); why do you prefer to use an enum?

Reply:
Sorry, I didn't mean to suggest that your reasoning was inadequate, more that I just disagree. I think supporting additional types of questions is better addressed with future versions of the schema and the enum version is a little more self-documenting and obvious.

Reply:
I changed it.